src/Importify/Cabal/Target.hs

```diff
@@ -23,34 +22,30 @@ module Importify.Cabal.Target
        , targetIdDir
        ) where
 
-import           Universum                       hiding (fromString)
-
-import           Data.Aeson                      (FromJSON (parseJSON),
-                                                  FromJSONKey (fromJSONKey),
-                                                  FromJSONKeyFunction (FromJSONKeyTextParser),
-                                                  ToJSON (toJSON), ToJSONKey (toJSONKey),
-                                                  Value (String), object, withObject,
-                                                  withText, (.:), (.=))
-import           Data.Aeson.Types                (Parser, toJSONKeyText)
-import           Data.Hashable                   (Hashable)
-import qualified Data.HashMap.Strict             as HM
-import qualified Data.Text                       as T (split)
-import           Distribution.ModuleName         (ModuleName)
-import           Distribution.PackageDescription (Benchmark (..), BenchmarkInterface (..),
-                                                  BuildInfo (..), CondTree,
-                                                  Executable (..),
-                                                  GenericPackageDescription (..),
-                                                  Library (..), TestSuite (..),
-                                                  TestSuiteInterface (..), condTreeData)
+import Universum hiding (fromString)
+
+import Data.Aeson (FromJSON (parseJSON), FromJSONKey (fromJSONKey),
+                   FromJSONKeyFunction (FromJSONKeyTextParser), ToJSON (toJSON),
+                   ToJSONKey (toJSONKey), Value (String), object, withObject, withText, (.:), (.=))
+import Data.Aeson.Types (Parser, toJSONKeyText)
+import Data.Hashable (Hashable)
```

Comment:
This is imported from `universum`

Reply:
Thanks! Probably I just forgot about this earlier... Or maybe older `universum` doesn't reexport this... But I want to merge this now so I will perform cleanup later...